apps.toml

```
@@ -386,6 +386,12 @@ state = "working"
 subtags = [ "business_and_ngos" ]
 url = "https://github.com/YunoHost-Apps/civicrm_drupal7_ynh"
 
+[cjdns]
+category = "system_tools"
+state = "working"
```

Is it really working? I will guess no

```suggestion
state = "notworking"
```

---

I see it failed on backup/restore and upgrade because I do not have those scripts. I thought those were optional. Do I need to have them or test only only the other ones?
Thanks